### ISSUE TITLE
Add alt text to user guide images

### ISSUE THREAD

**Issue by lauraykerr** [CDCgov/prime-simplereport#3998]
## Area or Component
User guide v 1.6 PDF

## Quick Summary
Most app screenshots in user guide are missing alt text.

## Accessibility Violation
Images in PDF should have alt text so they can be read by screen readers.

## Action Requested
Add descriptive alt text to all images — including screenshots and logos — to the source doc. 

## Source Code (optional)
[1.6 SimpleReport user guide](https://docs.google.com/document/d/1Hh3Gj4QOK2fVPNfrwd9nHSDO-waALcWHcHsGcHqudyM/edit?usp=sharing)

**Comment by lauraykerr** [comment]
Alt tags added to new version of guide and source Google doc
